types/office-js-preview/index.d.ts

```diff
@@ -83877,17 +83868,7 @@ declare namespace Word {
          * @param breakType Required. The break type to add to the body.
          * @param insertLocation Required. The value must be 'Start' or 'End'.
          */
-        insertBreak(breakType: Word.BreakType, insertLocation: Word.InsertLocation): void;
-        /**
-         * Inserts a break at the specified location in the main document.
-         *
-         * @remarks
-         * [Api set: WordApi 1.1]
-         *
-         * @param breakType Required. The break type to add to the body.
-         * @param insertLocation Required. The value must be 'Start' or 'End'.
-         */
-        insertBreak(breakType: "Page" | "Next" | "SectionNext" | "SectionContinuous" | "SectionEven" | "SectionOdd" | "Line", insertLocation: "Before" | "After" | "Start" | "End" | "Replace"): void;
+        insertBreak(breakType: Word.BreakType | "Page" | "Next" | "SectionNext" | "SectionContinuous" | "SectionEven" | "SectionOdd" | "Line", insertLocation: Word.InsertLocation.start | Word.InsertLocation.end | "Start" | "End"): void;
```

Comment:
Have we checked how readable this is in IntelliSense?

Reply:
Not yet. I'm now working on generating the reference docs